### PR TITLE
Remove Prototype

### PR DESCRIPTION
**Pull request by oliverguenther**
Replaces the prototype overview using Angular.

https://community.openproject.com/work_packages/23893/activity

### CODE REVIEW

File: app/views/my_projects_overviews/index.html.erb [hunk 1]
```diff
@@ -31,8 +30,8 @@ See doc/COPYRIGHT.md for more details.
   <% if User.current.allowed_to?(:edit_project, project) %>
     <li class="toolbar-item" title="<%= t(:label_personalize_page) %>">
       <%= link_to my_projects_overview_path(project), class: 'button', accesskey: accesskey(:edit) do %>
-        <i class="icon-settings"></i>
-        <span class="hidden-for-sighted"><%= t(:label_personalize_page) %></span>
+        <i class="icon-settings icon-context"></i>
+        <%= t(:button_edit_this_page) %>
```
Comment (ulferts) [inline app/views/my_projects_overviews/index.html.erb]: Why this change? I actually liked only having the gear wheel. Now there is so much to read.

**Comment by ulferts** [comment]
Files selected to be uploaded via a teaser element are uploaded all right when the user clicks on "save". However, they are only shown after a reload.

**Comment by ulferts** [comment]
This button is lying:

image

It will not take you back to the overview page. You will stay on the current page.

File: app/views/my_projects_overviews/index.html.erb (hunk 1 above)
Comment (oliverguenther) [inline app/views/my_projects_overviews/index.html.erb]: It was my personal preference. When I changed it, my argumentation was that:

1. we don't have toolbar items without text elsewhere (but I stand corrected, since Repositories have that as well)
2. I've removed the functionality to edit blocks directly on the index page, so I wanted to make it more clear where to edit this page

I have no strong opinion on that, though.

**Comment by oliverguenther** [comment]
I've addressed your points:

1. The button has been reduced to the cog item again
2. The page layout button is now `Save`.
3. Attachments are being reloaded explicitly through an additional request. I did not implement a check whether attachments need to be reloaded.

Should be good to go then.

**Comment by ulferts** [comment]
👍
